### ISSUE TITLE
Print Statistics

### ISSUE THREAD

**Issue by chraibi** [JuPedSim/jpscore#314]
In Gitlab by @gjaeger on Jun 17, 2019, 10:42 [[origin](https://gitlab.version.fz-juelich.de/jupedsim/jpscore/issues/324)]

from [Simulation.cpp, line 417 ff](https://gitlab.version.fz-juelich.de/jupedsim/jpscore/blob/develop/Simulation.cpp):
```c++
void Simulation::PrintStatistics(double simTime)

    for (const auto& itr : _building->GetAllTransitions()) {
        Transition* goal = itr.second;
        if (goal->GetDoorUsage()) {
            Log->Write(
                    "\nExit ID [%d] used by [%d] pedestrians. Last passing time [%0.2f] s",
                    goal->GetID(), goal->GetDoorUsage(),
                    goal->GetLastPassingTime());

            string statsfile = "flow_exit_id_"+to_string(goal->GetID())+".txt";
            if(goal->GetOutflowRate() <  (std::numeric_limits<double>::max)())
            {
                 char tmp[50];
                 sprintf(tmp, "%.2f", goal->GetOutflowRate());
                 statsfile = "flow_exit_id_"+to_string(goal->GetID())+"_rate_"+tmp+".txt";
            }
            Log->Write("More Information in the file: %s", statsfile.c_str());
            {
                 auto statOutput = new FileHandler(statsfile.c_str());
                 statOutput->Write("#Simulation time: %.2f", simTime);
                 statOutput->Write("#Flow at exit "+goal->GetCaption()+"( ID "+to_string(goal->GetID())+" )");
                 statOutput->Write("#Time (s)  cummulative number of agents \n");
                 statOutput->Write(goal->GetFlowCurve());
                 statOutput->FileHandler::~FileHandler();
            }

        }
    }
```

Can we change the filename `flow_exit_id_` in line [430](https://gitlab.version.fz-juelich.de/jupedsim/jpscore/blob/develop/Simulation.cpp#L430) like the trajectory filename (e.g. `flow_exit_id_filename.txt`?

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Jun 17, 2019, 19:43

mentioned in commit daa95675e69f6f11df15ef479d45235814101797

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Jun 18, 2019, 11:27

Added the name of the original trajectory file in the name of the flow file.

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Jun 18, 2019, 11:27

closed

**Comment by chraibi** [comment]
In Gitlab by @gjaeger on Jun 18, 2019, 20:56

It works fine. The files are stored in the folder of the ini-file. 

Can we define the folder like for trajectory files?

**Comment by chraibi** [comment]
In Gitlab by @gjaeger on Jun 18, 2019, 20:56

reopened

**Comment by schroedtert** [comment]
Can this be closed? See #649
